analytic_engine/src/meta/details.rs

```diff
@@ -1189,7 +1189,8 @@ mod tests {
         runtime.block_on(async move {
             let table_id = ctx.alloc_table_id();
             let partition_info = Some(PartitionInfo::Hash(HashPartitionInfo {
-                definitions: vec![Definition {
+                version: 0,
```

Comment:
Use `DEFAULT_PARTITION_VERSION` instead of magic number.

Reply:
Here is the test, I set a variable `default_version`.